pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -146,6 +146,14 @@
     )
     private String advertisedAddress;
 
+    //
```

Comment:
nit: remove this line if it is not needed.

Reply:
I have remove the code for this line